lib/web/websocket/websocket.js

```diff
@@ -184,10 +192,10 @@ class WebSocket extends EventTarget {
     }
 
     // 3. Run the first matching steps from the following list:
-    if (this[kReadyState] === WebSocket.CLOSING || this[kReadyState] === WebSocket.CLOSED) {
+    if (isClosing(this) || isClosed(this)) {
       // If this's ready state is CLOSING (2) or CLOSED (3)
       // Do nothing.
-    } else if (!isEstablished(this)) {
+    } else if (isConnecting(this)) {
```

Comment:
"      // If the WebSocket connection is not yet established"

Reply:
I am currently investigating this. Thats why is set it to draft.

We have 4 states. Connecting, Open, Closing, Closed. !isEstablished equals Connecting, Closing and Closed. Just because we checked before the cases for closing and closed we can use !isEstablished, as it reduces !isEstablished to Connecting. 

Tbh. feels super strange to not check explicitly for the right state.

Reply:
but isConnecting is

// If the WebSocket connection is not yet established, and the connection
  // is not yet closed, then the WebSocket connection is in the CONNECTING state.

Reply:
I prefer the name of isEstablished because it's what the spec uses. If isEstablished = isConnecting that's fine

Reply:
In this case